mozilla/blurts-server
# MNTOR-2125: only mark subscriber breach as resolved when all emails are marked as fixed

# References: 
Jira: MNTOR-2125
Figma: 




# Description
The correct behavior should be:
* when any 0-1/2 common breaches is fixed in the old UI, the new UI should still show the breach as active
* only when 2/2 common breaches are fixed in the old UI, the new UI should reflect the breach as being fixed

This bug is not going to be relevant when the redesign launches, as users won’t have an option to select an email address anymore, but I’ll put in a fix for it for now

# Screenshot (if applicable)

Not applicable.

# How to test
1.  On the current version resolve one of the common breaches on the primary account;
2. Refresh the redesigned version of the dashboard and observe the “Fixed” tab;
3. Go back to the current dashboard and uncheck the previously resolved breach;
4. Select the secondary email from the current dashboard;
5. Proceed to fix one of the common breaches;
6. Refresh the redesigned dashboard and observe the “Fixed” tab
7. Fix the common breach for both emails
8. Observe in new UI 

# Checklist (Definition of Done)
